src/main/kotlin/uk/gov/justice/digital/hmpps/personrecord/service/cprdomainevents/listeners/ClusterEventListener.kt

```diff
@@ -0,0 +1,59 @@
+package uk.gov.justice.digital.hmpps.personrecord.service.cprdomainevents.listeners
+
+import org.springframework.context.ApplicationEventPublisher
+import org.springframework.context.event.EventListener
+import org.springframework.scheduling.annotation.Async
+import org.springframework.stereotype.Component
+import org.springframework.transaction.event.TransactionalEventListener
+import uk.gov.justice.digital.hmpps.personrecord.service.cprdomainevents.events.cluster.BrokenCluster
+import uk.gov.justice.digital.hmpps.personrecord.service.cprdomainevents.events.cluster.OverrideConflict
+import uk.gov.justice.digital.hmpps.personrecord.service.cprdomainevents.events.cluster.SelfHealed
+import uk.gov.justice.digital.hmpps.personrecord.service.cprdomainevents.events.review.ReviewRaised
+import uk.gov.justice.digital.hmpps.personrecord.service.cprdomainevents.events.review.ReviewRemoved
+import uk.gov.justice.digital.hmpps.personrecord.service.cprdomainevents.events.telemetry.RecordClusterTelemetry
+import uk.gov.justice.digital.hmpps.personrecord.service.type.TelemetryEventType
+
+@Component
+class ClusterEventListener(
```

Comment:
We already have a PersonKeyEventListener - are Clusters different from PersonKeys? We have some work to do on naming